Description: Include forgotten headers
Author: Konstantinos Margaritis <markos@debian.org>

--- autobahn-cpp-17.5.1+git7cc5d37.orig/CMakeLists.txt
+++ autobahn-cpp-17.5.1+git7cc5d37/CMakeLists.txt
@@ -62,6 +62,8 @@ set(PUBLIC_HEADERS
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_procedure.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_publication.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_publication.ipp
+    ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_publish_options.hpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_publish_options.ipp
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_rawsocket_transport.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_rawsocket_transport.ipp
     ${CMAKE_CURRENT_SOURCE_DIR}/autobahn/wamp_register_request.hpp
